spec/app/domain/authentication/authn-jwt/create_identity_provider_spec.rb

```diff
@@ -30,15 +30,17 @@ def identity_available?
   let(:account) { 'my-account' }
 
   let(:authentication_parameters_url_identity) {
-    Authentication::AuthnJwt::AuthenticationParameters.new(Authentication::AuthenticatorInput.new(
-      authenticator_name: authenticator_name,
-      service_id: service_id,
-      account: account,
-      username: "dummy_identity",
-      credentials: "dummy",
-      client_ip: "dummy",
-      request: "dummy"
-    ))
+    Authentication::AuthnJwt::AuthenticationParameters.new(
+      authentication_input: Authentication::AuthenticatorInput.new(
+        authenticator_name: authenticator_name,
+        service_id: service_id,
+        account: account,
+        username: "dummy_identity",
+        credentials: "dummy",
+        client_ip: "dummy",
+        request: "dummy"
+      ),
+      jwt_token: nil)
```

Comment:
```suggestion
      jwt_token: nil
    )
```

Reply:
please do this for the rest of the files

Reply:
Done in all places